ktor-http/common/src/io/ktor/http/URLParser.kt

```diff
@@ -41,6 +41,30 @@ internal fun URLBuilder.takeFromUnsafe(urlString: String): URLBuilder {
     val slashCount = count(urlString, startIndex, endIndex, '/')
     startIndex += slashCount
 
+    if (protocol.name == "file") {
+        require(slashCount == 3)
+        host = urlString.substring(startIndex, endIndex)
```

Comment:
Why do we say that the path is a host while the host is actually missing (empty)?